src/backup/BackupPhrase.tsx

```diff
@@ -201,6 +216,7 @@ const styles = StyleSheet.create({
   },
   continueButton: {
     paddingHorizontal: variables.contentPadding,
+    paddingBottom: variables.contentPadding,
```

Comment:
For an iPhone 8, the continue button was on the bottom edge of the screen. Adding this to fix it.